Add min/max ranges to lots more of the commands

peternewman opened the pull request.

peternewman committed:
Add min/max ranges to lots more of the commands

magik1969 commented:
Please do not use value name if it is single value for command and value name match command name.
It's unnecessary redundancy. I try to avoid this when possible.
Use:
```
<command name="contrast" code="0x24" mode="rw">
        <!-- Set/get display contrast. Return number. Command: 'contrast 0-100', 'contrast?' -->
        <value min="0" max="100"/>
</command>
```

peternewman commented:
Ah, I was trying to simplify things for the consumer so every value has a name, rather than just some of them and needing to special-case stuff in my processing code.

peternewman committed:
Remove 'name' attribute from value elements

Where there is only one value in the command and hence it could be deemed unnecessary

peternewman committed:
Merge pull request #1 from peternewman/improvements-no-excess-names

Remove 'name' attribute from value elements

peternewman commented:
I've finally got round to fixing this if you'd like to take a look @magik1969 .

I'd also note if you're worried about the total size of the file/package in bytes, a lot of the comment section for each command is now redundant and could be removed, just leaving a high level description of what the command does (which could possibly be better placed in a "description" attribute or similar, so it could be parsed by machine too.

peternewman committed:
Merge branch 'magik1969:main' into improvements
